Web/Edubase.Web.UI/Areas/Governors/Models/GovernorsGridViewModel.cs

```diff
@@ -109,7 +109,9 @@ public GovernorsGridViewModel()
         private void CreateGrids(GovernorsDetailsDto dto, IEnumerable<GovernorModel> governors, bool isHistoric,
             int? groupUid, int? establishmentUrn)
         {
-            var roles = dto.ApplicableRoles.Where(role =>
+            var allRoles = governors.Select(g => (GR) g.RoleId).Distinct();
+
+            var roles = allRoles.Where(role =>
```

Comment:
If we remove the `applicableRoles` completely, the sections which say something to the effect of "there are no appointees for this role" will not show (instead, the headings will display only if there is a record of that type recorded).

This isn't necessarily a bad thing, but it is a change in behaviour.

To retain current behaviour, we could union with the applicable roles perhaps?

Reply:
think all this is fixed now